_aggregations/metric/maximum.md

```diff
@@ -9,30 +9,46 @@ redirect_from:
 
 # Maximum aggregations
 
-The `max` metric is a single-value metric aggregations that returns the maximum value of a field.
+The `max` metric is a single-value metric that returns the maximum value of a field.
 
-The following example calculates the maximum of the `taxful_total_price` field:
+The `max` aggregation compares numeric fields using a `double` (double-precision) representation. Results should be considered approximate for fields containing `long` or `unsigned_long` integer values greater than 2<sup>53</sup> because the number of significant bits in a `double` mantissa is 53.
+{: .note}
+
+## Parameters
+
+The `max` aggregation takes the following parameters.
+
+| Parameter | Required/Optional | Data type      | Description |
+| :--       | :--               | :--            | :--         |
+| `field`   | Required          | String         | The name of the field for which the maximum is computed.    |
+| `missing` | Optional          | Numeric        | The value to assign missing instances of the field. If not given, documents with missing values are omitted from the aggregation. |
+
+## Example
+
+This example finds the most expensive item---the item with the maximum value of the `base_unit_price`---in the e-commerce sample data:
 
 ```json
 GET opensearch_dashboards_sample_data_ecommerce/_search
 {
   "size": 0,
   "aggs": {
-    "max_taxful_total_price": {
+    "max_base_unit_price": {
       "max": {
-        "field": "taxful_total_price"
+        "field": "products.base_unit_price"
       }
     }
   }
 }
 ```
 {% include copy-curl.html %}
 
-#### Example response
+## Example response
+
+The response is as follows:
```

Comment:
@dwelsch-esi Let's rewrite to include more context (what does the response demonstrate to the user?).

Reply:
```suggestion
As shown in the following example response, the aggregation returns the maximum value of `max_base_unit_price`.
```

Reply:
@dwelsch-esi Terminating colon instead of a period to introduce the example, please. Otherwise LGTM.

Reply:
```suggestion
As shown in the following example response, the aggregation returns the maximum value of `products.base_unit_price`:
```